prod/test/test_algos.py

```diff
@@ -0,0 +1,208 @@
+    assert time_check # check the length of clips is <= clip_length
+
+    assert calc_result == answer
+
+def test_algo2(med_file):
+    min_=0.5
+    calc_result = algo2.run(
+        data=med_file,
+        min_=min_,
+        limit=10,
+        save_json=False
+    )
+
+    answer = [{'startTime': 14840.294, 'endTime': 14840.294},
+              {'startTime': 696.512, 'endTime': 696.512},
+              {'startTime': 6441.765, 'endTime': 6441.765},
+              {'startTime': 8194.165, 'endTime': 8194.165},
+              {'startTime': 8381.339, 'endTime': 8381.339},
+              {'startTime': 14140.362, 'endTime': 14140.362},
+              {'startTime': 22983.785, 'endTime': 22983.785},
+              {'startTime': 14428.961, 'endTime': 14428.961},
+              {'startTime': 15187.424, 'endTime': 15187.424},
+              {'startTime': 7199.836, 'endTime': 7201.808}]
+    time_check = check_length(calc_result, min_=min_)
+    assert time_check # check the length of clips is <= clip_length
+
+    assert calc_result == answer
+    
+
+def test_algo3_0(med_file):
+    min_=0.5
+    calc_result = algo3_0.run(
+        data=med_file,
+        min_=min_,
+        limit=10,
+        min_words=10,
+        save_json=False
+    )
+    answer = [{'startTime': 19827.532, 'endTime': 19857.415},
+              {'startTime': 12184.113, 'endTime': 12212.453},
+              {'startTime': 17847.854, 'endTime': 17877.303},
+              {'startTime': 5611.102, 'endTime': 5632.707},
+              {'startTime': 5919.616, 'endTime': 5949.445},
+              {'startTime': 6656.985, 'endTime': 6680.851}]
+    time_check = check_length(calc_result, min_=min_)
+    assert time_check # check the length of clips is <= clip_length
+
+    assert calc_result == answer
+
+def test_algo3_5(med_file):
+    min_=0.5
+    calc_result = algo3_5.run(
+        data=med_file,
+        min_=min_,
+        limit=10,
+        goal='num_emo',
+        save_json=False
+    )
+    answer = [{'startTime': 105.78, 'endTime': 135.449},
+              {'startTime': 234.249, 'endTime': 259.889},
+              {'startTime': 183.68, 'endTime': 200.871},
+              {'startTime': 375.218, 'endTime': 399.649},
+              {'startTime': 74.219, 'endTime': 102.902},
+              {'startTime': 266.303, 'endTime': 284.56},
+              {'startTime': 0.0, 'endTime': 10.329},
+              {'startTime': 135.939, 'endTime': 163.39},
+              {'startTime': 308.506, 'endTime': 334.573},
+              {'startTime': 338.925, 'endTime': 349.935}]
+    time_check = check_length(calc_result, min_=min_)
+    assert time_check # check the length of clips is <= clip_length
+
+    assert calc_result == answer
+    
+def test_algo1_diffs(med_file):
+    '''
+    Test different values for min_ result in different answers.
+    Does NOT check for correctness of those answers
+    '''
+    all_results = []
+    for time in [0.5, 3, 1, 2]:
```

Comment:
I would like to condense this using parametrization (issue #34) but for this PR I'm not touching it for now.